flow/core/params.py

```diff
@@ -304,6 +306,9 @@ def add(self,
             # FIXME: depends on simulator
             lane_change_params = SumoLaneChangeParams()
 
+        if energy_model not in ENERGY_MODELS:
```

Comment:
I think here we should also add a print statement like print([energy model] is not currently supported)